dev/benchmark.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: docu
+title: Benchmark Suite
+---
+
+## TODO: Document the .benchmark Format
+
+- preamble
+name
+group
+storage
+require
+
+load
+- loading instructions
+
+run
+- this is measured
+
+
+
+#### Clone the repo and compile the benchmark runner
+
+```
+git clone https://github.com/duckdb/duckdb
+cd duckdb
+BUILD_BENCHMARK=1 BUILD_TPCH=1 make
```

Comment:
Potentially it could be handy mentioning LTO='thin' here. Should work on clang or gcc.

Reply:
maybe `BUILD_TPCDS` as well?